turbo.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "tasks": {
+    "//#build": {
```

Comment:
Is there a reason these tasks are named "//..." but the task in https://github.com/isographlabs/isograph/pull/258/files is named "compile"?

Reply:
Name without # refers to script in every package, for single package it can be vite-demo#build. // Is special alias for root package.